packages/animations/lib/src/open_container.dart

```diff
@@ -74,13 +74,13 @@ typedef ClosedCallback<S> = void Function(S data);
 ///  * [Transitions with animated containers](https://material.io/design/motion/choreography.html#transformation)
 ///    in the Material spec.
 @optionalTypeArgs
-class OpenContainer<T extends Object> extends StatefulWidget {
+class OpenContainer<T extends Object?> extends StatefulWidget {
```

Comment:
What does T extends Object? do for the generic. Does this mean that T can be a class that can be null ?

Reply:
I tried removing this, but the analyzer complained with:

```
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at example/lib/container_transition.dart:241:29 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:24:18 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:197:18 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:366:16 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:541:16 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:715:16 • (implicit_dynamic_type)
  error • Missing type arguments for generic type 'OpenContainer<dynamic>' at test/open_container_test.dart:739:16 • (implicit_dynamic_type)
...
```